### ISSUE TITLE
I see no difference in PHPStorm after clicking "Configure PHPStorm"

### ISSUE THREAD

**Issue by slim16165** [getflywheel/local-addon-xdebug-phpstorm#1]
There is no log, no way to check how it should work. How do I understand if it's working?

**Comment by Luc45** [comment]
Yeah, nothing is happening to me as well. I'm trying to debug this line but I'm having a hard time:

```
    console.log(publicCWD.exists('./.idea/php.xml'));
    console.log(publicCWD.exists('./.idea/runConfigurations/Local-By-Flywheel.xml'));
```

It's not writing to the files.

**Comment by Luc45** [comment]
I got it. We have to normalize the slashes in the dir path. It is trying to find `~/` and replace it with the home dir, but in my case it is `~\`, so the replacement never occurs and the file doesn't write.

### How to fix:

1. Open FlyWheel
2. Go to Addons -> Installed -> Xdebug +PhpStorm -> Show Folder
3. Navigate to local-addon-xdebug-phpstorm/lib
4. Edit renderer.js
5. Replace line 21~:

```
const sitePath = site.path.replace('~/', userHome + '/').replace(/\/+$/, '') + '/';
```

With:

```
const sitePath = site.path.replace('\~\\', '~/').replace('~/', userHome + '/').replace(/\/+$/, '') + '/';
```

6. Reload Flywheel, either from Develop -> Refresh Window or by re-starting it.
7. Re-start PHPStorm

**Comment by paulotruta** [comment]
This one works like a charm for Windows environments.
@Luc45 Would be lovely if you could try to send this upstream in a way that works for both Windows and Mac!

Thanks for the tip!

**Comment by Luc45** [comment]
@paulotruta it should work with both Mac and Windows, since all it does is replace Windows's `~/` with Unix's `~\`, normalizing it. The rest is unchanged.
